src/PushNotifications.Projections/Subscriptions/SubscriberByTokenProjection.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Runtime.Serialization;
+using System.Threading.Tasks;
+using Elders.Cronus;
+using Elders.Cronus.Projections;
+using PushNotifications.Subscriptions;
+using PushNotifications.Subscriptions.Events;
+
+namespace PushNotifications.Projections.Subscriptions
+{
+    [DataContract(Name = "34e16311-85b7-4b4e-a7ca-a7004dd07d84")]
```

Comment:
Namespace

Reply:
In PushNotifications, all contracts don't have a namespace, so I followed the pattern.

Reply:
Also, it doesn't matter because this projection was removed in the next commit.